ptf_tests/tests/Nt_Ct_unsupported_arguments_values.py

```diff
@@ -0,0 +1,147 @@
+        ptf.dataplane_instance = ptf.dataplane.DataPlane(config)
+        self.capture_port = test_params['pci_bdf'][:-1] + "1"
+        self.config_data = get_config_dict(config_json, test_params['pci_bdf'])
+        self.gnmicli_params = get_gnmi_params_simple(self.config_data)
+        self.interface_ip_list = get_interface_ipv4_dict(self.config_data)
+
+
+    def runTest(self):
+        if not test_utils.gen_dep_files_p4c_dpdk_pna_ovs_pipeline_builder(self.config_data):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail("Failed to generate P4C artifacts or pb.bin")
+        
+        if not gnmi_cli_set_and_verify(self.gnmicli_params):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail("Failed to configure gnmi cli ports")
+
+        ip_set_ipv4(self.interface_ip_list)
+
+        # get port list and add to dataplane
+        port_list = self.config_data['port_list']
+        port_list[0] = test_utils.get_port_name_from_pci_bdf(self.capture_port)
+        port_ids = test_utils.add_port_to_dataplane(port_list)
+        
+        for port_id, ifname in config["port_map"].items():
+            device, port = port_id
+            self.dataplane.port_add(ifname, device, port)
+
+        if not ovs_p4ctl.ovs_p4ctl_set_pipe(self.config_data['switch'], self.config_data['pb_bin'], self.config_data['p4_info']):
+            self.result.addFailure(self, sys.exc_info())
+            self.fail("Failed to set pipe")
+
+        table = self.config_data['table'][2]
+        print(f"Rule Creation : {table['description']}")
+        print(f"Adding {table['description']} rules")
+        print("Expecting fail for Negative Test scenario:")
+        for match_action in table['match_action']:
+            if not ovs_p4ctl.ovs_p4ctl_add_entry(table['switch'],table['name'], match_action):
+                   print("-----------------------------------------------------")
+                   print("Above Fail is  as expected for Negative Test Exection")
+                   print("-----------------------------------------------------")
+           
+
+        table = self.config_data['table'][3]
+        print(f"Rule Creation for Negative Test case : {table['description']}")
+        print(f"Adding {table['description']} rules")
+        print("Expecting fail for Negative Test scenario:")
+        print(f"{table['match_action']}")
+        for match_action in table['match_action']:
+            print(match_action)
+            if not ovs_p4ctl.ovs_p4ctl_add_entry(table['switch'],table['name'], match_action):
+                   print("-----------------------------------------------------")
+                   print("Above Fail is  as expected for Negative Test Exection")
+                   print("-----------------------------------------------------")
+                   
+                                      
+
+        # Sleep 5 Sec before tcp connection establishment
+        time.sleep(5)
```

Comment:
not required. as no tcp handshake happening

Reply:
updated